cmd/kubectl-gadget/deploy.go

```diff
@@ -128,6 +131,21 @@ func init() {
 		"debug", "d",
 		false,
 		"show extra debug information")
+	deployCmd.PersistentFlags().StringVar(
+		&socketPathContainerd,
+		"socketpath-containerd",
+		"",
```

Comment:
I think we should put here the default value we are using to let users know what path we are currently using. For instance, for Docker ["/run/docker.sock"](https://github.com/kinvolk/inspektor-gadget/blob/5b93ab18b3b22741b26ae326d259156928d0cd54/pkg/container-utils/docker/docker.go#L34).

Reply:
Had thought about that as well. What i wanted was to reference the `DefaultSocketPath` from `container-utils/docker/docker.go` where the default socket path was already defined, since i didn't want to have 2 lines with default socket paths.
Problem was that it introduced a circular import and so it did not compile :)

So we would also need to move that default socketpath definition from `docker.go` (same for crio and containerd) to another independent file, right?

Reply:
Mmmmm I don't see what is the circular import 🤔